### DISCUSSION TITLE
Call pest test within an other test

### DISCUSSION THREAD

**Discussion by abrardev99** [pestphp/pest#576] [Q&A]
Is there any way to call a pest test another test? Our current PHPUnit tests have some tests, which are called other PHPUnit tests inside. For example

```php
function somethingTest(){
    ...
}


function firstTest(){
    $this->somethingTest()
}
 
function secondTest(){
    // this test implementation start

    $this->somethingTest()

    // this test implementation start
}
```

Currently, We are using pest helpers for this purpose. There is another option [depends](https://github.com/pestphp/pest/pull/103), but this does not suit the use case. We sometimes need to call the test in the center of other tests, but I think it `depends` executes it first.

**Comment by fabio-ivona** [comment]
pest helpers are a good solution to execute repeated complex tests

another way is to use custom expectations

**Reply by abrardev99** [reply]
I see. Does this worth a feature if I'd send PR?

**Reply by fabio-ivona** [reply]
could you elaborate your idea?
